### DISCUSSION TITLE
Welcome to ConFIG Discussions!

### DISCUSSION THREAD

**Discussion by qiauil** [tum-pbs/ConFIG#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wrongww** [comment]
Hi, thank you for this very impressive work.
I am just wondering if there is a missing arg in your example code:
    g_config=ConFIG_update(grads) 
    apply_gradient_vector(network)  # shouldn't be apply_gradient_vector(network, **g_config**) ?

**Reply by qiauil** [reply]
Hi, @wrongww 

Yes! that is a typo. Thanks for pointing it out; I have fixed it!

Feel free to post if you find any other issues.

Cheers
